### PR TITLE
Goal: rename `--allow-extra-opcode-budget` to `--allow-more-opcode-budget`

### PR DESCRIPTION
**Pull request by ahangsu**
## Summary

For the sake of uniformity, we rename `--allow-extra-opcode-budget` to `--allow-more-opcode-budget`, to align with `--allow-more-logging`.

### CODE REVIEW

File: cmd/goal/clerk.go [hunk 1]
```diff
@@ -69,10 +69,10 @@ var (
...
 	requestOutFilename string
 
-	simulateAllowEmptySignatures   bool
-	simulateAllowMoreLogging       bool
-	simulateAllowExtraOpcodeBudget bool
-	simulateExtraOpcodeBudget      uint64
+	simulateAllowEmptySignatures  bool
+	simulateAllowMoreLogging      bool
+	simulateAllowMoreOpcodeBudget bool
+	simulateExtraOpcodeBudget     uint64
```
Comment (algoanne) [inline cmd/goal/clerk.go:75]: didn't want to also change this one?

Comment (ahangsu) [reply cmd/goal/clerk.go:75]: I want to slightly push back on changing this one, for I want the boolean flags (for convenience of extra budget) to align with `--allow-more-logging`, but for specifying a particular budget, we stay the same.

Comment (algoanne) [reply cmd/goal/clerk.go:75]: ok, makes sense.

**Comment by bbroder-algo** [comment]
idk...

**Review by jasonpaulos — APPROVED** [review]
Since we don't need to change the oas2 this is a relatively small change, so seems fine to me.

**Comment by ahangsu** [comment]
we missed the boat... adios

**Review by jannotti — APPROVED** [review]

**Pull request merged**